lib/DashPlatformProtocol.js

```diff
@@ -16,10 +16,13 @@ class DashPlatformProtocol {
    * @param {Object} options
    * @param {StateRepository} [options.stateRepository]
    * @param {JsonSchemaValidator} [options.jsonSchemaValidator]
+   * @param {boolean} [options.identities.enableLockTxOneBlockConfirmationFallback]
    */
   constructor(options = {}) {
     this.stateRepository = options.stateRepository;
     this.jsonSchemaValidator = options.jsonSchemaValidator;
+    this.enableLockTxOneBlockConfirmationFallback = options.identities
```

Comment:
Do we need this property or we could just pass the option to `StateTransitionFacade`?

Reply:
Right now we don't need it